dali/pipeline/operator/copy_vector_helper.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATOR_COPY_VECTOR_HELPER_H_
+#define DALI_PIPELINE_OPERATOR_COPY_VECTOR_HELPER_H_
+
+#include <type_traits>
```

Comment:
I'd say this thing belongs to `dali/core`

Reply:
Done